test/mysql.discover.test.js

```diff
@@ -271,4 +271,129 @@ describe('Discover and build models', function() {
       done();
     });
   });
+
+  describe('discoverModelProperties() flags', function() {
+    describe('with default flags', function() {
```

Comment:
It's definitely subjective, but we documented the convention we decided to go with at https://github.com/strongloop/loopback-contributor-docs/blob/master/style-guide.md#good-12